.github/workflows/build-scala-cli-template.yml

```diff
@@ -20,10 +20,18 @@ jobs:
         with:
           fetch-depth: 0
       - name: Install CIRCT
-        uses: circt/install-circt@v1.0.0
+        id: install-circt
+        if: ${{ inputs.circt != 'default' }}
+        uses: circt/install-circt@v1.1.1
         with:
           version: ${{ inputs.circt }}
           github-token: ${{ github.token }}
+      # TODO have install-circt do this
```

Comment:
I agree that this seems reasonable to do.

That would avoid this type of chained conditional steps. Such chained conditional steps usually seems like an anti-pattern. No real problem with this as-is, though.